packages/path_provider/path_provider_linux/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.0
+* This release updaes getApplicationSupportPath to use the application ID instead of the executable name.
+  * Older apps will lose their preferences as they migrate to the new app ID based path.
```

Comment:
It's not preferences, it's any data stored by previous versions under getApplicationSupportPath. Lot of packages use path_provider.